### PR TITLE
Print first letter of every word in the string

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Print first letter of every word in the string

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

I want to add the code in c++ in cp.
#5482

### CODE REVIEW

File: C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+Description : In a given string, the task is to create a string
+              with the first letter of every word in the string.
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: Quite vague. Can you be more descriptive?

File: C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+/*
+Test Cases :
+    Test Case 1 :
+        Input : united public school
+        Output : ups
+    Test Case 2 : 
+        Input : computer science
+        Output : cs
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: Add a new line at the EOF

File: C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                val = false;
+            }
+        }
+        return result;
+    }
+};
+
+int main()
+{
+    //input string
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: Redundant Comment

File: C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    string input_string;
+    cout << "Enter the string : " << endl;
+    getline(cin, input_string);
+    //object of a class
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: Redundant Comment

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp (hunk 1 above)
Comment (anshmishra010) [reply C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: It's a question present in gfg , and they also provided the same description ,
You can check the link - https://practice.geeksforgeeks.org/problems/print-first-letter-of-every-word-in-the-string3632/1

Comment (anshmishra010) [reply C-Plus-Plus/cp/Print_first_letter_of_every_word_in_the_string.cpp]: Still I will try to explain it with example

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**